org.osgi.test.junit5/bnd/afterAll.bnd

```diff
@@ -0,0 +1 @@
+Export-Package: ${p}.tb1.*;-split-package:=first
```

Comment:
seems odd to have all these embedded jars have the exact same contents, no? Why no reuse just one?

Reply:
You can't install multiple bundles with the same bsn at the same time. In the multi-level test there are several different bundles installed at any given time.

Reply:
Where would you like them? In the bnd files or in the pom? Or both?

Reply:
The comment about it being opaque is fair. I will try and add some comments to explain the whole thing.

Reply:
I added comments in the .bnd files that refer to the appropriate place in the code where they are used, and then explained in the code how/why they are used.